connectors/src/connectors/google_drive/temporal/workflows.ts

```diff
@@ -68,6 +72,25 @@ export async function googleDriveFullSync({
     foldersToBrowse = await getFoldersToSync(connectorId);
   }
 
+  setHandler(folderUpdatesSignal, (folderUpdates: FolderUpdatesSignal[]) => {
+    // If we get a signal, update the workflow state by adding/removing folder ids.
+    for (const { action, folderId } of folderUpdates) {
+      switch (action) {
+        case "added":
+          foldersToBrowse!.push(folderId);
```

Comment:
Why the `!` ? I think we disallow this

Reply:
foldersToBrowse is declared undefined but if it's undefined we'll populate it with `    foldersToBrowse = await getFoldersToSync(connectorId);` (line 72) it can't be undefined at that line but typescript does complain.
Do you prefer that I rewrite it with a wrapping "if" ?

Reply:
Yes no `!` please 🙏

Reply:
This should be forbidden by linter. In front it is IIRC